### PR TITLE
Moved Interop  Locations For System.IO.Compression.Brotli

### PR DESCRIPTION
**Pull request by eanova**
This commit is to relocate the project's interop definitions into the Common library project.

### CODE REVIEW

**Comment by eanova** [comment]
According to the stacktraces both checks fail during testing of System.Text.Json.Tests,   not quite sure what the next step would be.  Is there a way to rerun the tests?

File: src/System.IO.Compression.Brotli/src/System.IO.Compression.Brotli.csproj [hunk 1]
```diff
@@ -9,8 +9,8 @@
   <!-- Default configurations to help VS understand the options -->
   <!-- Platform agnostic files -->
   <ItemGroup>
-    <Compile Include="Interop\Interop.Brotli.Encoder.cs" />
-    <Compile Include="Interop\Interop.Brotli.Decoder.cs" />
+    <Compile Include="$(CommonPath)\Interop\Interop.Brotli.Encoder.cs" />
+    <Compile Include="$(CommonPath)\Interop\Interop.Brotli.Decoder.cs" />
```
Comment (stephentoub) [inline src/System.IO.Compression.Brotli/src/System.IO.Compression.Brotli.csproj]: I'd prefer to see all of the interop signatures for Brotli in a single file, especially if it's going to just be sitting in the root Interop folder.

Comment (eanova) [reply src/System.IO.Compression.Brotli/src/System.IO.Compression.Brotli.csproj]: Ok, I'll move everything into a single "Interop.Brotli.cs" file.

**Comment by ahsonkhan** [comment]
> According to the stacktraces both checks fail during testing of System.Text.Json.Tests, not quite sure what the next step would be. Is there a way to rerun the tests?

https://github.com/dotnet/corefx/issues/36618

I've run them again. (you can use the azp command): https://github.com/dotnet/corefx/blob/34118b74d46129b3095e1b4dbad8441c4db62e33/Documentation/project-docs/pullrequest-builds.md#azure-pipelines-commands

**Comment by eanova** [comment]
Thanks @ahsonkhan , it's good to know these kind of test errors can occur between different builds.

**Pull request merged**